gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -1822,6 +1824,8 @@ gum_exec_ctx_ensure_helper_reachable (GumExecCtx * ctx,
   write (ctx, cw);
 
   gum_arm64_writer_flush (cw);
+  gum_clear_cache (slab->data, gum_arm64_writer_offset (cw));
```

Comment:
This may clear the wrong area, because the writer was initialized with `slab->data + slab->offset` and not `slab->data`. Use `cw.base` instead.